.circleci/config.yml

```diff
@@ -209,7 +209,7 @@ jobs:
             fi
   edge_lambdas_build:
     docker:
-      - image: circleci/node:8.11.3
```

Comment:
Wondering if we should pin this to 8.10 instead?

> AWS Lambda supports the following Node.js runtimes.
> 
> Node.js Runtimes
> 
> Name | Identifier
> -- | --
> Node.js 8.10 | nodejs8.10
> Node.js 6.10 | nodejs6.10
> 
> 

And would be in line with [what we have here](https://github.com/wellcometrust/wellcomecollection.org/blob/master/cache/edge_lambdas.tf#L29)

Reply:
As it's where the tests are running, this makes loads of sense to stop us using features that aren't available in `8.10`.

Reply:
To be less opaque:

> Wondering if we should pin this to 8.10 instead?

Yip! Great idea.